doc/build-windows-mingw.md

```diff
@@ -0,0 +1,260 @@
+Building Bitcoin on Native Windows
+==================================
+
+This document describes the installation of the mysys and mingw tools as well as 
+the build dependencies and then finally the compilation of the bitcoin binaries.
+
+NOTE: This script will only prepare a build system and bitcoin in 32 bit.
+
+Prepare your build system
+-------------------------
+
+1.1 Install msys shell:
+http://sourceforge.net/projects/mingw/files/Installer/mingw-get-setup.exe/download
+From MinGW installation manager -> All packages -> MSYS
+mark the following for installation:
+
+msys-base-bin
+msys-autoconf-bin
+msys-automake-bin
+msys-libtool-bin
+
+Make sure no mingw packages are checked for installation or present from a previous install. Only the above msys packages should be installed. Also make sure that msys-gcc and msys-w32api packages are not installed.
+
+then click on Installation -> Apply changes
+
+
+1.2 Install MinGW-builds project toolchain:
+Download http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.9.2/threads-posix/dwarf/i686-4.9.2-release-posix-dwarf-rt_v3-rev1.7z/download
+and unpack it to C:\
+
+1.3. Ensure that mingw-builds bin folder is set in your PATH environment variable. On Windows 7 your path should look something like:
+
+
+C:\mingw32\bin;%SystemRoot%\system32;%SystemRoot%;%SystemRoot%\System32\Wbem;%SYSTEMROOT%\System32\WindowsPowerShell\v1.0\
+
+
+1.4 Additional checks:
+C:\MinGW\bin should contain nothing but mingw-get.exe.
+Your gcc -v output should be:
```

Comment:
Since we haven't mentioned either a Windows CMD shell or the MSYS shell yet, we should probably mention which one to use here, as both will work.

NOTE: The msys shell will match your output exactly, the CMD shell will match everything except the second output which will be `COLLECT_GCC=gcc` instead of listing the full path.